ispdb/inbox.lv

```diff
@@ -2,25 +2,49 @@
 <clientConfig version="1.1">
   <emailProvider id="inbox.lv">
     <domain>inbox.lv</domain>
-    <!-- Same as inbox.lt TODO merge? -->
-    <displayName>Inbox.lv</displayName>
-    <displayShortName>Inbox.lv</displayShortName>
+    <domain>inbox.eu</domain>
+    <domain>inbox.lt</domain>
+    <domain>inbox.la</domain>
+    <displayName>%EMAILDOMAIN%</displayName>
+    <displayShortName>%EMAILDOMAIN%</displayShortName>
+    <incomingServer type="imap">
+      <hostname>mail.%EMAILDOMAIN%</hostname>
+      <port>993</port>
+      <socketType>SSL</socketType>
+      <authentication>password-cleartext</authentication>
+      <username>%EMAILADDRESS%</username>
+    </incomingServer>
     <incomingServer type="pop3">
-      <hostname>mail.inbox.lv</hostname>
+      <hostname>mail.%EMAILDOMAIN%</hostname>
       <port>995</port>
       <socketType>SSL</socketType>
-      <username>%EMAILLOCALPART%</username>
+      <username>%EMAILADDRESS%</username>
       <authentication>password-cleartext</authentication>
       <pop3>
         <leaveMessagesOnServer>true</leaveMessagesOnServer>
       </pop3>
     </incomingServer>
     <outgoingServer type="smtp">
-      <hostname>mail.inbox.lv</hostname>
+      <hostname>mail.%EMAILDOMAIN%</hostname>
       <port>587</port>
       <socketType>STARTTLS</socketType>
```

Comment:
Please use SSL and 465 here. This matches the incoming servers.
Same for the other configs below.